inc/admin.php

```diff
@@ -275,6 +275,7 @@ function enqueue_admin_scripts( $prefix = '', $force = false ) {
 				'directory_enabled'         => ! empty( $directory_enabled ),
 				'tutorials_enabled'         => ! empty( $tutorials_enabled ),
 				'copy_content'              => siteorigin_panels_setting( 'copy-content' ),
+				'cache'						=> array(),
```

Comment:
I was thinking that, but I actually prefer having it loaded slightly later rather. There are quite a few interfaces that use Page Builder, and having all the tutorial content always in the header might be a bit heavy.